### PR TITLE
Fe/lab3

### PR DESCRIPTION
**Pull request by KirylAkulich1**
https://kirylakulich1.github.io/FrontEndTasks/fe_lab3/gamerules.html

### CODE REVIEW

File: fe_lab3/commands.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    <script src="js/main.js"></script>
+</head>
+<body>
+    <div class="container">
+         <nav>
+            <ul class="site-nav">
+                <li class="site-nav__item"><a class="site-nav__ref" href="index.html">Main</a></li>
+                <li class="site-nav__item"><a class="site-nav__ref" href="gamerules.html">Game Rules</a></li>
+                <li class="site-nav__item site-nav__item-right">
+                    <a>Theme</a>
```
Comment (gmltA) [inline fe_lab3/commands.html:26]: Why do you need `a` here?

File: fe_lab3/gameproc.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            <div class="game-info">
+                <span class="team-name">Team1</span>
+                <label>You scrore </label>
+                <span class="guessed-score">999</span>
+            </div>
+            <div class="game-word-area">
+                <span class="guess-word">word to guess</span>             
+            </div>
+            <div class="game-info bottom">
+                <label>Skipped</label>
```
Comment (gmltA) [inline fe_lab3/gameproc.html:21]: Why `label` here?

**Review by gmltA — COMMENTED** [review]
That's a <kbd>6</kbd>. See requirements if you are willing to improve it.

**Comment by KirylAkulich1** [comment]
Hmmm...Can I proceed doing other labs and in future update this mark?

**Comment by gmltA** [comment]
> Hmmm...Can I proceed doing other labs and in future update this mark?

Actually no. This ark will be fixed as soon as I hit 'approve'. But we can figure something out as an exception in the future, but I'm not promising you anything.

**Comment by KirylAkulich1** [comment]
I agree with 6️⃣

**Review by gmltA — APPROVED** [review]
So be it!

**Pull request merged**